presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Add support for get partition names by filter.
+* Add forms of approx_percentile accepting an accuracy parameter.
+* Improve error handling for geometry deserialization edge cases.
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix ScanFilterAndProjectOperator raw input bytes accounting for LazyBlocks.
+* Improve read-path to better estimate the size of row. Output (sum) string stat even if min/max values are too long.
+* Improve PrestoS3FileSystem to work with the AWS Default Credentials Provider.
+* Improve optimizer performance for array_join.
```

Comment:
I don't think this change is about the optimizer performance.  I think it's just optimizing the performance of array_join.  (https://github.com/prestodb/presto/pull/13874) 

`Improve performance of array_join`.